diff --git a/luci.mk b/luci.mk
--- a/luci.mk
+++ b/luci.mk
@@ -140,7 +140,7 @@ ifeq ($(PKG_NAME),luci-base)
 
        menu "Translations"$(foreach lang,$(LUCI_LANGUAGES),$(if $(LUCI_LANG.$(lang)),
 
          config LUCI_LANG_$(lang)
-           tristate "$(shell echo '$(LUCI_LANG.$(lang))' | sed -e 's/^.* (\(.*\))$$/\1/') ($(lang))")
+           tristate "$(shell echo '$(LUCI_LANG.$(lang))' | sed -e 's/^.* (\(.*\))$$/\1/') ($(lang))"$(if $(filter pl,$(lang)),\n\t\t\tdefault y,))
 
        endmenu
      endef
